src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
Ihor recommended specifying the name of the transition that you apply the duration on. For example `transition: color 300ms, transform 300ms;` At the moment you have` transition-property: all` (set by default because you didn't specify it); and `transition-duration: 300ms` applied to all; This is not ideal.